### ISSUE TITLE
About the featuremetric alignment.

### ISSUE THREAD

**Issue by Haonan-DONG** [cvg/pixel-perfect-sfm#2]
Thanks for the excellent work on Structure from Motion.
One question for the paper:
In the feature extraction, are the features extracted from the pre-trained CNNs? Since I found no loss function to train the CNNs.

**Comment by Skydes** [comment]
Thanks for your question. We indeed use pre-trained CNNs. Most experiments are conducted with S2DNet (Germain et al. ECCV 2020) but we also experiment with other weights in the ablation study (Table 4).
